fix: Remove unread messages notification

artonge opened the pull request:
This is unnecessarily spamming the view when the chat tab is closed. Also, this is kind of a redundant with the blue dot on the message icon.

B | A
-- | --
<img width="293" height="59" alt="image" src="https://github.com/user-attachments/assets/49bf170e-f3bb-4e70-9c7c-9b4fdb0d32f1" /> | <img width="392" height="54" alt="2025-09-09_16h46_25" src="https://github.com/user-attachments/assets/eab88579-787b-46ab-8236-9833847d7892" />

Antreesy reviewed (commented):
I don't mind personally. Maybe we can make 'blue dot' more prompting, e.g. flashing with red/blue or opacity (like recording icon)? cc @nimishavijay

artonge committed:
fix: Remove unread messages notification

This is unnecessarily spamming the view when the chat tab is closed.
Also, this is kind of a duplicate with the blue dot on the message icon.

Signed-off-by: Louis <louis@chmn.me>

nimishavijay commented:
I think the toast can often be useful for those with the chat closed. I don't think getting rid of this and fully relying on the tiny dot is a good idea, but I do get that the current version can be improved: 

- [ ] A simple improvement would be to style it differently. We can give it a dark background, make it look different from the regular toast styling, and position it somewhere else with an animation and a shorter timeout, so it doesn't feel like "that annoying thing that doesn't go away"
- [ ] if we do want to remove it, then we could show a counter bubble next to the conversation icon with the number of unread messages, and maybe a soft shadow or something around that button, that would change every time there is a new message and bring attention to itself 
- [ ] A way to make the toast more useful is to show the messages themselves in that little toast, but I'm guessing it is out of scope for now.
